stacks/knative/uninstall.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+set -e
+
+OPERATOR_VERSION="1.2.2"
+
+# Remove Knative Serving and Eventing resources first
+kubectl delete KnativeServing knative-serving -n knative-serving
+kubectl delete KnativeEventing knative-eventing -n knative-eventing
```

Comment:
should we also add here this here?

```shell
kubectl delete ns knative-serving
kubectl delete ns knative-eventing
```

Reply:
Thanks for pointing out. Added required steps in latest commit.